src/App.js

```diff
@@ -1,9 +1,33 @@
 import './App.css';
+import TrocadilhoList from './components/TrocadilhoList/TrocadilhoList';
 
 function App() {
+  const trocadilhos =[{date: "13/02/2023",
+  votes: 10,
+  dev: "cicero.medeirios",
+  context: "Lorem ipsum dolor,sit alit. Fugiat animi quaerat assumenda architecto possimus qui itaque reprehenderit tempore provident debitis? Dicta blanditiis mollitia reiciendis saepe. Alias, facere? Excepturi, officia obcaecati.",
+  message: "Lorem ipsus"
+  },{date: "13/02/2023",
+  votes: 10,
+  dev: "cicero.medeirios",
+  context: "Lorem ipsum dolor,sit alit. Fugiat animi quaerat assumenda architecto possimus qui itaque reprehenderit tempore provident debitis? Dicta blanditiis mollitia reiciendis saepe. Alias, facere? Excepturi, officia obcaecati.",
+  message: "Lorem ipsus"
+  },{date: "13/02/2023",
+  votes: 10,
+  dev: "cicero.medeirios",
+  context: "Lorem ipsum dolor,sit alit. Fugiat animi quaerat assumenda architecto possimus qui itaque reprehenderit tempore provident debitis? Dicta blanditiis mollitia reiciendis saepe. Alias, facere? Excepturi, officia obcaecati.",
+  message: "Lorem ipsus"
+  },{date: "13/02/2023",
+  votes: 10,
+  dev: "cicero.medeirios",
+  context: "Lorem ipsum dolor,sit alit. Fugiat animi quaerat assumenda architecto possimus qui itaque reprehenderit tempore provident debitis? Dicta blanditiis mollitia reiciendis saepe. Alias, facere? Excepturi, officia obcaecati.",
+  message: "Lorem ipsus"
+  },] 
```

Comment:
Please think of this as an entry data, that can be fetch from an API endpoint. Move this to another place to keep this file clean and easy read.